packages/velo-external-db-core/lib/service/schema_aware_data.js

```diff
@@ -7,13 +7,16 @@ class SchemaAwareDataService {
         this.dataService = dataService
     }
 
-    async find(collectionName, filter, sort, skip, limit) {
+    async find(collectionName, filter, sort, skip, limit, _projection) {
```

Comment:
currently we don't send projection from router, so why add this param here ?
we didn't change the api, so currently the projection is just an internal thing, please remove this.